File: lists/fib.py


def fib():
    result = [1, 1]
    result_str = []
    while len(result) < 20:
        temp = result[-2] + result[-1]
        temp_str = "{} + {} = {}".format(result[-2], result[-1], temp)
        result.append(temp)
        result_str.append(temp_str)

    for i in result_str:
        print(i)


fib()

File: lists/fib.js


const fib = () => {
    let nums = [1, 1];
    let strings = [];

    while (strings.length < 20){
        let temp = nums[nums.length - 2] + nums[nums.length - 1];
        strings.push(`${nums[nums.length - 2]} + ${nums[nums.length - 1]} = ${temp}`);
        nums.push(temp);
    }

    for (let i in strings) {
        console.log(strings[i]);
    }
}

fib();